nilearn/_utils/glm.py

```diff
@@ -156,19 +156,17 @@ def make_stat_maps(
 
         .. versionadded:: 0.9.2
 
-    first_level_contrast : :obj:`str` or :class:`numpy.ndarray` of \
-                        shape (n_col) with respect to \
-                        :class:`~nilearn.glm.first_level.FirstLevelModel` \
-                        or None, default=None
+    %(first_level_contrast)s
+
         For :class:`~nilearn.glm.second_level.SecondLevelModel`,
         in case a :obj:`list` of
-        :class:`~nilearn.glm.first_level.FirstLevelModel` was provided
-        as ``second_level_input``,
+        :class:`~nilearn.glm.first_level.FirstLevelModel`
+        or a dataframe was provided as ``second_level_input``,
         we have to provide a :term:`contrast`
         to apply to the first level models
         to get the corresponding list of images desired,
         that would be tested at the second level.
```

Comment:
This part already exists in `nilearn/_utils/docs`, non? Or I am missing a detail.